### DISCUSSION TITLE
Works with my LogiTech headset but not my Focusrite

### DISCUSSION THREAD

**Discussion by kjcole** [jamulussoftware/jamulus#3033] [Technical Support & Help]
Running an up-to-date **Pop! OS LTS (22.04)** and **Jamulus 3.9.1** and **Pipewire**.

When I route my **Focusrite Scarlett 2i2 "Camera" Analog Stereo capture** into Jamulus and then Jamulus into **Focusrite Scarlette 2i2 "Camera" Analog Stereo playback**, it plays back through  **Built-in Audio Analog Stereo** instead.

It works okay with the **Logi Zone 750**, but I'd like it to use the nicer microphone and headphones connected to the Focusrite...

**NOTE**: The **Analog** capture on the **Logi Zone 750** doesn't work at all, but the **Digital** does. (And there's no Digital playback.)

Other applications seem to manage with the Focusrite okay -- though sometimes after a while, some applications seem to lose the Focusrite -- but they don't switch to built-in when that happens.

I'm hoping to stick with the LTS releases of as much as possible.

I also have **Helvum** installed but I'm not using it.  

(I *just now* tried **Patchance** because I saw it recommended in a search for similar discussions, but, alas it fails as well.)

**Comment by ann0see** [comment]
I don't know pipewire enough - but can you edit config files/use qjackctl to control the JACK integration? I'd start looking for a way to change the in/output devices there. 
Otherwise I'd look in the Arch Wiki - they're quite good for the "new" linux stuff. 

Or https://github.com/flathub/io.jamulus.Jamulus
